zboxcore/sdk/filerefsworker.go

```diff
@@ -106,13 +114,15 @@ func (o *ObjectTreeRequest) getFileRefs(oTR *oTreeResponse, bUrl string) {
 	}
 	oResult := ObjectTreeResult{}
 	ctx, cncl := context.WithTimeout(o.ctx, time.Second*30)
+	respStatusCode := 500
 	err = zboxutil.HttpDo(ctx, cncl, oReq, func(resp *http.Response, err error) error {
 		if err != nil {
 			l.Logger.Error(err)
 			return err
 		}
 		defer resp.Body.Close()
 		respBody, err := ioutil.ReadAll(resp.Body)
+		respStatusCode = resp.StatusCode
```

Comment:
Move code one line above, we should process the error right after it is returned, please do not insert any code that is not necessary there.